src/utils/evm/test-routes.ts

```diff
@@ -0,0 +1,339 @@
+
+import { EVMAssetTransfer, EVMGenericMessageTransfer, Environment, EthereumConfig, EvmResource, ResourceType, SubstrateConfig } from '@buildwithsygma/sygma-sdk-core'
+import { BigNumber, Wallet, providers, utils } from 'ethers'
+import { RpcEndpoints } from '../../types'
+
+import { KeyringPair } from '@polkadot/keyring/types'
+import { Storage__factory } from '../Contracts'
+import { ApiPromise, WsProvider } from '@polkadot/api'
+import { AccountInfo } from '@polkadot/types/interfaces'
+import { LoggingData, fetchTokenAmount, sleep, waitUntilBridgedFungibleEvm } from '../helpers'
+import { print } from 'gluegun'
+
+const executionContractAddresses = {
+  EXECUTE_CONTRACT_ADDRESS_1:
+    '0xdFA5621F95675D37248bAc9e536Aab4D86766663',
+  EXECUTE_CONTRACT_ADDRESS_2:
```

Comment:
Why are these hardcoded?
And if they are hardcoded shouldn't it be by network?

Reply:
It is per network. the 1, and 2 indicating the domainID. On cronos and base currently we are not able to do generic transfers, because we are not able to calculate the fee. The addresses are hardcoded to minimize the possibility of misuse of the tool.